### PR TITLE
use category images for og:image on category page

### PR DESCRIPTION
**Pull request by dosbiner**

### CODE REVIEW

File: templates/_partials/microdata-jsonld.tpl [hunk 1]
```diff
@@ -35,6 +35,8 @@
             <meta property="og:image" content="{$p_img.large.url}"/>
         {/foreach}
     {/if}
+{elseif $page.page_name == 'category'}
```
Comment (NeOMakinG) [inline templates/_partials/microdata-jsonld.tpl]:
```suggestion
{elseif $page.page_name === 'category'}
```

Prefer using strict operators :)

**Comment by prestarocket** [comment]
@micka-fdz what do you think about this PR ?

**Comment by micka-fdz** [comment]
I think we need to test if image exists before using it.
But I don't know if category images are relevant here.
@NeOMakinG what do you recommend?

File: templates/_partials/microdata-jsonld.tpl (hunk 1 above)
Comment (NeOMakinG) [reply templates/_partials/microdata-jsonld.tpl:38]: Also, check us image exist before, overtwhise you'll end with some og:image metadata with a wrong url, which will lead to a bug on social networks

**Comment by NeOMakinG** [comment]
@micka-fdz @prestarocket It depends of your own choice to be honest, if you prefer that every shops get their logo shown on social networks shares or the category image then close this PR, otherwise if you think that showing the category img is right, then do it! Anyway it needs to check is the category image exist, and if not, then showing the shop logo.

**Pull request closed without merging**